aries-site/src/pages/templates/forms.mdx

```diff
@@ -42,6 +43,7 @@ Forms don't need to be a tedious task for users to complete, as they often come
 ### Required vs. optional fields
 
 - When possible, restrict forms only to fields that are required in order to keep them concise.
+- For single form fields (ex: delete confirmation request), use an asterisk to denote that it is a "required" field . This can avoid user confusion and reduce the chance of "required" input error.
```

Comment:
```suggestion
- For [single form fields](#single-required-formfield) (ex: delete confirmation request), use an asterisk to denote that it is a "required" field. This can avoid user confusion and reduce the chance of "required" input error.
```